compose/config/config.py

```diff
@@ -171,7 +172,7 @@ def version(self):
         if version == '2':
             version = V2_0
 
-        if version != V2_0:
+        if version not in (V2_0, V2_1):
```

Comment:
minor: maybe `if version == V1` would be more correct here, since it's the only version that would not be supported (now and in the future).